src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx

```diff
@@ -98,3 +103,14 @@ const AccommodationsForm = (props: any): JSX.Element => {
 };
 
 export default AccommodationsForm;
+
+// Temporary fix until sharedBathroom is changed into a boolean
+function isSharedBathroom(input: string): boolean {
+  if (!input) return false;
+  if (parseInt(input) && (parseInt(input) !== 0)) {
+    return true;
+  }
+  if (input.length > 10) return true;
+  const normalizedInput = input.toLowerCase();
+  return normalizedInput === 'yes';
+}
```

Comment:
ran scripts on prod data to test:

![image](https://user-images.githubusercontent.com/18321302/51064176-34c27800-15b3-11e9-86df-08f5523d3afa.png)

Reply:
The Lower Bedroom Bath is sort of a shared because Liviing room and Kitchen is down there  : true

🤣

This looks reasonable as a temporary fix. The code didn't make sense to me until I saw the examples from production; maybe amend the comment above with a note like "this is specific to values in existing listings" or similar?

Also (I'm just code golfing here) I think this is equivalent to:

    return !!input && (parseInt(input) > 0 || input.length > 10 || input.toLowerCase() === 'yes');

Reply:
Can we manually replace `The lower bathroom is sort of....` with a ``true`` or ``yes`` in the database? I think it would be better if this was a `yes` and the disclaimer be left in the description or other house notes.

Reply:
should we manually add it into the house notes or make a note to the host about it?

Reply:
i manually changed `The lower bathroom...` to `Yes` in the prod db.